release-process.md

```diff
@@ -319,4 +319,18 @@ Once everything is working (website docs, website changes) create an announcemen
 and then send an e-mail to the mailing list. To create an announcement, create a post under
 `news/_posts` and then run `jekyll build`.
 
+<h4>Update a test suite depending on old releases</h4>
+
+Since Spark always keeps the latest maintance released of each branch in the mirror network,
```

Comment:
maintance released -> maintenance release
and
with downloading -> by testing against (?)

Reply:
Thanks!